src/local/butler/reproduce_tool/config.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Configuration helpers for the reproduce tool."""
+
+import os
+
+from base import json_utils
+from base import memoize
+from base import utils
+from local.butler.reproduce_tool import errors
+from local.butler.reproduce_tool import http_utils
+from system import shell
+
+CONFIG_DIRECTORY = os.path.join(
+    os.path.expanduser('~'), '.config', 'clusterfuzz')
+AUTHORIZATION_CACHE_FILE = os.path.join(CONFIG_DIRECTORY, 'authorization-cache')
+CONFIG_URL_FILE = os.path.join(CONFIG_DIRECTORY, 'config-url')
+
+
+def set_configuration_url(config_url):
+  """Configure the tool to connect to the specified domain."""
```

Comment:
Seems incomplete, it should add, "to fetch the configuration and store it locally."

Reply:
No longer relevant in the current patch set